### DISCUSSION TITLE
About the two parameters, fhlwr and fhswr, in RRTMG scheme.

### DISCUSSION THREAD

**Discussion by lo-y-wni** [ufs-community/ufs-srweather-app#730] [Q&A] [answered]
Hello everyone:

I want to confirm the function of the two parameters, fhlwr and fhswr, is to control the calling frequency of RRTMG? Is it like the `radt` in WRF but the unit is in second?

Yi-Chuan

**Comment by EdwardSnyder-NOAA** [comment, marked as answer]
Good afternoon Yi-Chuan,

The `fhlwr` and `fhswr` variables do control the calling frequency for the longwave and shortwave radiation based on the radiation scheme of the CCPP suite you are using. These variables are similar to the `radt` variable in WRF, with the exception of `radt` controls both shortwave and longwave radiation calls and is defined in minutes instead of seconds.

Please let us know if you have any additional questions.

Thanks!

Eddie Snyder | Advanced User Support (AUS) Team
